data/human/sales.txt

```diff
@@ -258,7 +258,8 @@ outfitter "Common Outfits"
 	"Pilot's License"
 	"Laser Rifle"
 	"Fuel Bar Fixed Display"
-
+	"Astronomical Scanner"
```

Comment:
```suggestion
```

I second the recommendation to remove it from the "common outfits" section. The Deep Sky outfitters is a good place to have it for now.